File: test/cs/test_DiffusionLimitedEvaporation.cs
'Test generation'

public static class Test
{
    //first);
    //test1

    public static void test1()
    {
        DiffusionLimitedEvaporation res0 = Estimation_DiffusionLimitedEvaporation.CalculateDiffusionLimitedEvaporation(deficitOnTopLayers:5341,soilDiffusionConstant:4.2);

        Console.WriteLine("{0}","diffusionLimitedEvaporation: "+res0.diffusionLimitedEvaporation);

        Console.WriteLine("diffusionLimitedEvaporation Comparison: ("+Math.Round( 6605.505,3)+";"+Math.Round(res0.diffusionLimitedEvaporation,3)+") "+Equals(Math.Round( 6605.505,3),Math.Round(res0.diffusionLimitedEvaporation,3)));
    }
}

File: test/java/test_DiffusionLimitedEvaporation.java
//Test generation'

import java.math.BigDecimal;
public static class Test
{
    //first);
    //test1);

    public static void test1()
    {
        double deficitOnTopLayers = 5341;
        double soilDiffusionConstant = 4.2;

        DiffusionLimitedEvaporation res0 = Estimation_DiffusionLimitedEvaporation.CalculateDiffusionLimitedEvaporation(deficitOnTopLayers,soilDiffusionConstant);
        System.out.println(" diffusionLimitedEvaporation: "+res0.diffusionLimitedEvaporation);

        System.out.println(((new BigDecimal(res0.diffusionLimitedEvaporation)).setScale(3, BigDecimal.ROUND_HALF_DOWN)).equals((new BigDecimal( 6605.505)).setScale(3, BigDecimal.ROUND_HALF_DOWN)));
    }
}
